libs/nss/patches/010-create_openwrt_target.patch

```diff
@@ -0,0 +1,126 @@
+--- /dev/null	2018-11-22 01:42:28.168002140 +0200
++++ b/nss/coreconf/OpenWRT.mk	2018-11-23 23:40:08.676708840 +0200
+@@ -0,0 +1,123 @@
++#
++# This Source Code Form is subject to the terms of the Mozilla Public
++# License, v. 2.0. If a copy of the MPL was not distributed with this
++# file, You can obtain one at http://mozilla.org/MPL/2.0/.
++
++include $(CORE_DEPTH)/coreconf/UNIX.mk
++
++RANLIB = ranlib
++CC = $(TARGET)
++DEFAULT_COMPILER= $(TARGET)
++CCC = $(TARGETCXX)
++CROSS_COMPILE = 1
++NATIVE_CC = gcc
++DEFINES += -DHAVE_POSIX_FALLOCATE=0  -D_REENTRANT
++OPTIMIZER = -Os -gdwarf-2
```

Comment:
If possible strip all additional flags and add use the TARGET_CFLAGS variable instead